composables/breadcrumbs.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { withTrailingSlash } from 'ufo'
+
+export const useBreadcrumbs = () => {
+  const currentPath = useRoute().path
+
+  const unfilteredPathSections = currentPath.split('/')
+
+  const currentPathSections = unfilteredPathSections.filter(function (
+    el: string
+  ) {
+    return el !== ''
+  })
```

Comment:
1. It is better here to call functions in chain
2. It is more readable if you pass arrow function into filter
3. Empty string is casted to boolean as `false`, so it will be more versatile to check element as `!!el`

```suggestion
  const currentPathSections = currentPath.split('/').filter((el) => !!el)
```